src/styles/components/_button.css

```
@@ -0,0 +1,44 @@
+.button{
+  color: var(--black);
+  border-color: var(--light-green);
+  background-color: var(--white);
+  border-style: solid;
+  border-radius: 10px;
+  width: 100px;
+  height: 30px;
+  margin: 5px;
+  cursor: pointer;
+  font-family: var(--font-base);
+  font-weight: bolder;
+}
+
+.button:disabled {
+  cursor: default;
+  opacity: 50%;
+}
+
+.button--general:hover{
+  background-color: var(--light-green);
+  color:var(--white);
+}
+
+.button--active, button--general:active{
```

Acho que aqui deveria ser classe né?
```suggestion
.button--active, .button--general:active{
```